### PR TITLE
Implement business model choice selection

### PR DESCRIPTION
**Pull request by Bachibouzouk**

### CODE REVIEW

**Comment by paulapreuss** [comment]
trying to look at these PR changes I get a `TemplateDoesNotExist: cp_nigeria/steps/b_models.html` error. Did you forget to commit this file (or rename it along the way)?

**Comment by Bachibouzouk** [comment]
> trying to look at these PR changes I get a `TemplateDoesNotExist: cp_nigeria/steps/b_models.html` error. Did you forget to commit this file (or rename it along the way)?

I added it now, as it was a new file I did not see it as I have too many files not in the repo (I should add them to gitignore)

File: app/cp_nigeria/views.py [hunk 1]
```diff
@@ -450,36 +410,44 @@ def cpn_model_choice(request, proj_id, step_id=6):
...
+        bm, created = BusinessModel.objects.get_or_create(
+            scenario=project.scenario, defaults={"scenario": project.scenario}
+        )
+        context["form"] = ModelSuggestionForm(instance=bm)
+        context["bm"] = bm
+        context["score"] = bm.total_score
+
+        recommended = context["form"].fields["model_name"].initial
+        if recommended is not None:
+            context["recommanded_model"] = recommended
```
Comment (paulapreuss) [inline app/cp_nigeria/views.py:425]:
```suggestion
            context["recommanded_model"] = recommended
            context["form"] = ModelSuggestionForm(instance=bm, initial={"model_name": recommended})
```

load the recommended business model as the initial value to the form, so the information for the suggested business models gets loaded immediately after saving the questionnaire (without having to select it from the drop down)

**Comment by paulapreuss** [comment]
I think this is a good way of implementing it. From a user perspective I think the questionnaire seems a bit long and it would maybe be best to break it up into sections instead of showing all at once, but let's see what feedback the consortium gives.

**Review by paulapreuss — APPROVED** [review]

**Pull request merged**